modules/machine-autoscaler-about.adoc

```diff
@@ -9,4 +9,14 @@ The MachineAutoscaler adjusts the number of Machines in the MachineSets that you
 deploy in an {product-title} cluster. You can scale both the default `worker`
 MachineSet and any other MachineSets that you create. The MachineAutoscaler
 makes more Machines when the cluster runs out of resources to support more
-deployments.
\ No newline at end of file
+deployments. The MachineAutoscaler also reconciles any changes that you make to
```

Comment:
> The MachineAutoscaler also reconciles any changes that you make to the MachineSets, such as changing the minimum or maximum number of instances.

I think this is a good addition, but just my $0.02, I might reword to make it clear that the thing being modified is the MachineAutoscaler since users should not be modifying MachineSets directly. Maybe something along the lines of:

"Any changes to the values in MachineAutoscaler resources are immediately applied to the MachineSet they target."